util/reggen/alert.py

```diff
@@ -11,7 +11,7 @@
 
 class Alert(Signal):
     def __init__(self, name: str, desc: str, bit: int, fatal: bool):
-        super().__init__(name, desc, Bits(bit, bit))
+        super().__init__(name, desc, Bits(bit, bit), False)
```

Comment:
I think you need to do something similar in `util/reggen/interrupt.py` to get this to pass CI.

Reply:
I replaced some code in interupt.py but i still got problem with flash cm:
```ERROR: The generated security countermeasures testplan /home/otdev/workspace/upstream/opentitan/hw/top_earlgrey/ip/flash_ctrl/data/autogen/flash_ctrl_sec_cm_testplan.hjson is stale. Please manually update it with the newly added (or removed) countermeasures.```